capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -61,7 +61,7 @@
                 null, null, 0,
                 0, foodAvailable, requiredFee),
       ExampleMatcher.matching().withIgnorePaths("datastoreId", "organizationId", "eventLatitude", "eventLongitude", "rank")
-      ), Sort.by(Sort.Direction.DESC, "rank")
+      ), Sort.by(Sort.Direction.DESC, "rank_")
```

Comment:
I don't think you meant to commit this -- it was a test to make sure Datastore failed on a nonexistent column.